video/out/hwdec/hwdec_drmprime.c

```diff
@@ -64,6 +65,17 @@ const static dmabuf_interop_init interop_inits[] = {
     NULL
 };
 
+/**
+ * Due to the fact that all the v4l2 support for SoCs exists in forked ffmpeg
+ * trees, the weird pixel formats they use are not in upstream ffmpeg and can't
```

Comment:
fwiw these seem to be rpi specific, as you know rockchip works out-of-the-box with a forked ffmpeg

Reply:
I wrote it this way because I'm expecting a wacky format to show up for the rockchip hevc decoder, but you're right that it's just rpi today.